DevEdu.API/DevEdu.Business.Tests/LessonServiceTests.cs

```diff
@@ -0,0 +1,195 @@
+﻿using DevEdu.Business.Services;
+using DevEdu.Business.ValidationHelpers;
+using DevEdu.DAL.Repositories;
+using Moq;
+using NUnit.Framework;
+
+namespace DevEdu.Business.Tests
+{
+    public class LessonServiceTests
+    {
+        private Mock<ILessonRepository> _lessonRepoMock;
+        private Mock<ICommentRepository> _commentRepoMock;
+        private Mock<IUserRepository> _userRepoMock;
+        private Mock<IUserValidationHelper> _userValidationHelperMock;
+        private Mock<ILessonValidationHelper> _lessonValidationHelperMock;
+
+
+        [SetUp]
+        public void Setup()
+        {
+            _lessonRepoMock = new Mock<ILessonRepository>();
+            _commentRepoMock = new Mock<ICommentRepository>();
+            _userRepoMock = new Mock<IUserRepository>();
+            _userValidationHelperMock = new Mock<IUserValidationHelper>();
+            _lessonValidationHelperMock = new Mock<ILessonValidationHelper>();
+        }
+
+        [Test]
+        public void AddLesson_SimpleDto_CreatedLesson()
+        {
+            //Given
+            var expected = LessonData.LessonId;
+
+            var lessonDto = LessonData.GetAddedLessonDto();
+
+            _lessonRepoMock.Setup(x => x.AddLesson(lessonDto)).Returns(expected);
+
+            var sut = new LessonService(_lessonRepoMock.Object, _commentRepoMock.Object, _userRepoMock.Object,
+                _userValidationHelperMock.Object, _lessonValidationHelperMock.Object);
+
+            //When
+            var actualId = sut.AddLesson(lessonDto);
+
+            //Then
+            Assert.AreEqual(expected, actualId);
+            _lessonRepoMock.Verify(x => x.AddLesson(lessonDto), Times.Once);
```

Comment:
what about lesson topics?
can we have them in lesson's input model? what do you think?